lib/bibliothecary/related_files_info.rb

```diff
@@ -20,9 +21,12 @@ def self.create_from_file_infos(file_infos)
     def initialize(file_infos)
       package_manager = file_infos.first.package_manager
       @platform = package_manager.platform_name
-      @folder_path = File.join(file_infos.first.folder_path, File.dirname(file_infos.first.relative_path))
+      file_infos.first.tap do |initial|
+        @folder_path = Pathname.new(initial.folder_path).join(File.dirname(initial.relative_path)).cleanpath.to_path
+        @relative_folder_path = Pathname.new(File.dirname(initial.relative_path)).cleanpath.to_path
+      end
```

Comment:
something about the naming here is confusing - a comment with examples would be pretty helpful I think.

Reply:
I'm pretty confused myself (also about fileinfo itself). I'm wondering if we should ditch both and just add a "path". I think the problem is that it's basing it on the root path of the project it's run in, and that doesn't seem useful if you're scanning a subpath.